weave-js/src/components/PagePanelComponents/Home/Home.tsx

```diff
@@ -251,6 +268,11 @@ const HomeComp: FC<HomeProps> = props => {
       <LayoutElements.Block>
         <HomeTopBar />
       </LayoutElements.Block>
+      {viewerIsEmployee && (
```

Comment:
I wonder if we want to have some convention (like a light background shading?) as a visual hint that what the employee is seeing is not what a customer would see. In general it would be good if we could quickly eliminate any employee-specific UI we introduce until we have something like an admin mode that we'd encourage devs to only use when they need it. Otherwise it's too easy to introduce something that seems fine to the developer but doesn't work well for user.

Reply:
Agreed - this is only supposed to be for a few days while we get the copy written